src/com/owncloud/android/lib/resources/activities/GetRemoteActivitiesOperation.java

```diff
@@ -59,7 +59,7 @@
     private static final String TAG = GetRemoteActivitiesOperation.class.getSimpleName();
 
     // OCS Routes
-    private static final String OCS_ROUTE = "/apps/activity/api/v2/activity";
+    private static final String OCS_ROUTE = "/ocs/v2.php/apps/activity/api/v2/activity";
```

Comment:
@MorrisJobke so much doubt in your words :P 

https://github.com/nextcloud/activity/blob/apiv2/docs/endpoint-v2.md

(You just need to look at the proper branch :P)